index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a-b});
+    this.length =+1;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items.includes(this.items[pos])){
+      return this.items[pos]
+    } else {
+      throw new Error('OutOfBounds')
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length>0) {
+      var max = this.items.reduce(function(a, b) {
+        return Math.max(a, b);
```

Comment:
Here you are doing the same thing twice: you are comparing and then returning the max value, and with the Math method you are returning the max value again. You should stick to one of these.